melos.yaml

```diff
@@ -24,5 +24,3 @@ scripts:
     exec: dcm analyze .
   format:
     exec: dart format --set-exit-if-changed .
-  test:
```

Comment:
Since example folders don't have a "test" folder and only in the root folder of the package, the `melos test` fails in the CI.  However, this is fine to remove since GitHub check also runs `flutter test` directly in the root package folder.